examples/create-setters/simple/README.md

```diff
@@ -0,0 +1,84 @@
+# create-setters: Simple Example
+
+### Overview
+
+Setters provide a solution for template-free setting of comments. The
+`create-setters` KRM config function applies comments to resource fields
+with setter references.
+
```

Comment:
In this example, we will see how to add setter comments declaratively to resource fields using `create-setters function`.